src/main/java/edu/cornell/kfs/sys/batch/service/DigesterXMLBatchInputFileType.java

```diff
@@ -0,0 +1,88 @@
+package edu.cornell.kfs.sys.batch.service;
+
+import java.io.ByteArrayInputStream;
+
+import org.apache.commons.digester3.Digester;
+import org.apache.commons.digester3.binder.DigesterLoader;
+import org.apache.commons.digester3.binder.RulesModule;
+import org.apache.commons.digester3.xmlrules.FromXmlRulesModule;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import java.net.URL;
+import javax.xml.validation.Schema;
+
+import org.kuali.kfs.sys.batch.XmlBatchInputFileTypeBase;
+import org.kuali.kfs.sys.exception.ParseException;
+import org.kuali.kfs.sys.exception.XmlErrorHandler;
+
+// CU customization to continue support for commons-digester after KualiCo has removed it from base code;
+// This can be removed once all code has been updated to use jaxb or other framework to replace commons-digester.
+public abstract class DigesterXMLBatchInputFileType<T> extends XmlBatchInputFileTypeBase {
```

Comment:
I generated the class name by adding Digester in front of the base class name. I see how it would make sense to change it to "Xml". Will update.

Will update the extend from.

The `outputClass` is not used by our Digester implementation but I will look to see if it makes sense to add it to the configuration for completeness. Thank you.